grafana/risingwave-dashboard.py

```diff
@@ -400,6 +405,11 @@ def section_streaming_actors(outer_panels):
                     "actor_sampled_serialize_duration_ns", "{{actor_id}}"
                 ),
             ]),
+            panels.timeseries_backpressured_percent("Actor Backpressure Rate", [
+                panels.target(
+                    "rate(stream_actor_output_buffer_blocking_duration[15s]) / 150000000", "{{actor_id}}"
```

Comment:
why / 150000000 ?

Reply:
15, 000, 000, 000 / 100 * 100%
Should doc it.

Reply:
Where is 15, 000, 000, 000 come from?

Reply:
Fixed.

Reply:
So why 15 now? `rate` function already gives you *per second* data. You cannot always make guesses and make the data look "reasonable" -- we need accurate and explainable things.

Reply:
My mistake for remembering incorrectly.